site.yml

```diff
@@ -20,3 +21,22 @@
     - role: caddy
     - role: xsnippet_api
     - role: xsnippet_web
+
+
+- hosts: localhost
```

Comment:
@malor Do you think we want this to be a part of this playbook? Or should these calls be a part of corresponding github workflows?

Reply:
I would just leave it here. I like that the playbook verifies that it has correctly configured the infrastructure.